core/modules/layout/layout.admin.inc

```diff
@@ -29,20 +29,18 @@ function layout_list_page() {
     $row = array();
     // Default layouts have a null path.
     if ($path === '') {
-      $row[] = array('data' => t('Default layouts (used on all paths without a specific layout)'));
-      // Empty columns are intentional so that they may be hidden with
-      // priority-low class on the $header array.
-      $row[] = '';
-      $row[] = '';
+      $row[] = array('data' => t('Default layouts (used on all paths without a specific layout)'), 'colspan' => '3');
```

Comment:
...turns out that my method works only when all columns are visible. The `colspan` value needs to dynamically change (JS?) to `2` as one column get hidden when the viewport gets smaller and then needs to be entirely removed when the only columns shown are the "Path and layout" and "Operations".

Beyond my skills, so anybody that wants to pick this up, feel free.